jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PushImageStepTest.java

```diff
@@ -89,6 +98,54 @@ public void testMakeListForManifestList() throws IOException, RegistryException
     }
   }
 
+  @Test
+  public void testMakeList_multiPlatform_enabled() throws IOException, RegistryException {
+    Image asd = Image.builder(V22ManifestTemplate.class).setArchitecture("wasm").build();
```

Comment:
```suggestion
    Image image = Image.builder(V22ManifestTemplate.class).setArchitecture("wasm").build();
```
or some name that reads better.